Makefile

```diff
@@ -47,13 +48,13 @@ lint:
 	  --rule 'max-len: [off]' \
 	  -- test
 	$(REMEMBER_BOWER) $(shell pwd)
-	@echo 'Checking for missing link definitions...'
-	grep -o '\[[^]]*\]\[[^]]*\]' index.js \
-	| sort -u \
-	| sed -e 's:\[\(.*\)\]\[\]:\1:' \
-	      -e 's:\[.*\]\[\(.*\)\]:\1:' \
-	      -e '/0-9/d' \
-	| xargs -I '{}' sh -c "grep '^//[.] \[{}\]: ' index.js"
+	rm -f README.md
+	VERSION=0.0.0 make README.md
+	$(REMARK) \
+	  --use remark-lint-no-undefined-references \
+	  --use remark-lint-no-unused-definitions \
+	  -- README.md
+	git checkout README.md
```

Comment:
The new approach is no shorter than the old approach, even though it contains no logic. :stuck_out_tongue_winking_eye: